python/python/glide/async_commands/server_modules/json.py

```diff
@@ -320,6 +320,47 @@ async def forget(
     )
 
 
+async def numincrby(
+    client: TGlideClient,
+    key: TEncodable,
+    path: TEncodable,
+    number: Union[int, float],
+) -> Optional[bytes]:
+    """
+    Increments or decrements the number of the JSON values at the specified `path' by `number` within the JSON document stored at `key`.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        path (TEncodable): The path within the JSON document.
+        number (Union[int, float]): The number to increment or decrement by.
+
+    Returns:
+        Optional[bytes]:
+            For JSONPath ('path' starts with '$'):
+                Returns a bytes string representation of an array of bulk strings that indicates the new values after incrementing for each matched 'path'.
+                If a value is not a number, its corresponding return value will be 'None'.
+                If `path` doesn't exist, an empty array will be returned.
+            For legacy 'path' (`path` doesn't start with `$`):
+                Returns a bytes string representation of the resulting value after the increment or decrement.
+                If multiple values are selected, the result of the last updated value is returned.
+                If the value at the 'path' is not a number or 'path' doesn't exist, an error is raised.
+            If 'key' does not exist, an error is raised.
+            If the result is out of the range of 64-bit IEEE double, an error is raised.
+
+    Examples:
+        >>> from glide import json
+        >>> await json.set(client, "doc", "$", '{"a": [], "b": [1], "c": [1, 2], "d": [1, 2, 3]}')
+        >>> await json.numincrby(client, "doc", "$.d[*]", 10)›
+            b'11,12,13'  # Increment each element in `d` array by 10.
```

Comment:
I am pretty sure this is wrong and supposed to be 
```suggestion
            b'[11,12,13]'  # Increment each element in `d` array by 10.
```